packages/nimble-components/src/wafer-map/modules/rendering.ts

```diff
@@ -1,34 +1,35 @@
+import type { WaferMap } from '..';
 import type { DieRenderInfo, Dimensions } from '../types';
-import type { DataManager } from './data-manager';
 
 /**
- * Responsible for drawing the dies inside the wafer map
+ * Responsible for drawing the dies inside the wafer map and scaling the canvas
  */
-export class RenderingModule {
+export class RenderingModule extends EventTarget {

```

Comment:
This does not get used as an event target in this PR so we should not make that change here.